example/aks/main.tf

```diff
@@ -0,0 +1,114 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.44.0"
+    }
+  }
+  required_version = ">=0.14.8"
```

Comment:
Seeing as how this is the examples section, suggest using the pessimistic operator (as in previous comment), e.g. `"~> 0.14"`.